proto/config/netinst.proto

```diff
@@ -107,4 +114,19 @@ message NetworkInstanceConfig {
 
   // static DNS entry, if we are running DNS/DHCP service
   repeated ZnetStaticDNSEntry dns = 41;
+  // Enable to use DHCP to automatically propagate routes for uplink subnets
+  // into applications connected to them indirectly through local network instances.
+  // This option is only valid for local network instances. For other types
+  // of network instances, it will be ignored.
+  bool propagate_connected_routes = 42;
```

Comment:
I always suspect that the question about life and everthing had something to do with IP routing ;-)

Reply:
Should be do this with 'disable_propagate_connected_routes'? since default case is most people want.

Reply:
I'm also considering renaming this to `propagate_uplink_routes`. After all, uplink port may receive additional routes from an external DHCP server. Shouldn't we propagate all the routes associated with an uplink interface?

Reply:
Not going to rename, we agreed internally that only connected routes will be propagated, not all routes received from external DHCP server.

Reply:
Makes sense to keep the name and semantics (need to set to true to change from current behavior).
@naiming-zededa Even if most cases (where app instances have multiple adapters) could benefit from having this set it is a bit scary to start doing that as part of an update by changing the default and have a disable field.

Reply:
Sure.